tools/gromacs/sim.xml

```diff
@@ -257,10 +265,12 @@
             <!-- <param name="ndx_bool" value="false" /> -->
             <param name="traj" value="trr"/>
             <param name="str" value="gro"/>
+            <param name="tpr_out" value="true"/>
```

Comment:
can you add num_outputs to the tests so that you test the filtering?

Reply:
Good idea, thanks!